lib/managers/files.js

```diff
@@ -1243,6 +1243,78 @@ Files.prototype.getCollaborations = function(fileID, options, callback) {
 };
 
 /**
+ * Enum of valid x-rep- hint values for generating representation
+ *
+ * @readonly
+ * @enum {FileRepresentationType}
```

Comment:
Where is this type defined?  You may need to put a `@typedef` somewhere to describe what this type is